scripts/uninstall.sh

```diff
@@ -13,18 +13,11 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 
-err() {
-  MESSAGE=$1
-  echo -e >&2
-  echo -e "\e[91mERROR: \e[37m$MESSAGE" >&2
-  echo -e >&2
-}
-
-versionNumber() {
```

Comment:
why you removed this? this function don't exist in lib

Reply:
It's there: https://github.com/dynatrace-oss/dynatrace-gcp-function/blob/embedded_ext_tools/scripts/lib.sh#L62